torch/_torch_docs.py

```diff
@@ -3922,6 +3922,35 @@ def merge_dicts(*dicts):
     tensor(7.5107e-16, dtype=torch.float64)
 """.format(**common_args))
 
+add_docstr(torch.isin, r"""
+isin(elements, test_elements, assume_unique=False, invert=False) -> Tensor
```

Comment:
Typically (but not always) PyTorch signatures start with "input" and use "other" for the second input tensor. NumPy's signatures, on the other hand, are all over the place. We do try to care about kwarg name consistency (even when we just expect an arg to be used as a kwarg), but in this case input and other can be assumed to be positional.

Reply:
I suggest making assume_unique and invert kwarg-only. That's more consistent with the array-api (see https://data-apis.github.io/array-api/latest/API_specification/index.html?highlight=isin). It is inconsistent with NumPy but that's OK.

Reply:
> Typically (but not always) PyTorch signatures start with "input" and use "other" for the second input tensor.

While I see your point, I think that the names `elements` and `test_elements` are less ambiguous for quickly understanding what order to pass inputs to `isin()`. Is consistency more important here?

> (see https://data-apis.github.io/array-api/latest/API_specification/index.html?highlight=isin)

I'm a bit confused because I don't see `isin()` in that link, but I made `assume_unique` / `invert` kwarg-only regardless.

Reply:
Sorry, the link is confusing because isin() isn't in the new array-api. The functions in the array-api, however, use positional-only and kwarg-only args aggressively.